src/routes/legal/privacy-policy.md

```diff
@@ -39,7 +39,6 @@ Often times you can clear your own data simply by deleting your account.
 
 As we said, you can request any GDPR/Privacy things in [this Matrix room](https://matrix.to/#/#gdpr:projectsegfau.lt). But, if there's any ongoing investigations and a law enforcement agency contacts us, we will comply. But, we will only do it if we can verify it's official and they have a warrant.
 
-
-We hope you can understand our privacy policy.
+Thank You for reading and we hope you can understand our privacy policy.
```

Comment:
I think it'd be better to remove this, as it sounds kind of stupid tbh.

Reply:
agree